airports.go

```diff
@@ -31,22 +46,38 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
-		if !flaga {
-			flaga = true
+		if !flag {
+			flag = true
 			continue
 		}
 		item := record[0]
 		itemLabel := record[1]
 		icao := record[2]
 		iata := record[3]
 		point := record[4]
+		if len(iata) !=3 {
+			iata=""
+		}
 		lat, lon := calculatePosition(point)
 		_, err1 := fmt.Fprintf(file1, "%v|%v|%v|%v|%v|%v\n", item, itemLabel, icao, iata, lat, lon)
+		record_counter++
 		if err1 != nil {
 			log.Fatal(err1)
 		}
-
+		
+		if last_icao == icao && !flag1{
+			duplicate_counter++
+			flag1 = true
+			_, err2 := fmt.Fprintf(file2, "%v|%v|%v|%v|%v|%v\n", item, itemLabel, icao, iata, lat, lon)
+			if err2 != nil {
+				log.Fatal(err2)
+			}
+		} else {
+			flag1 = false
+		}
```

Comment:
if icao is present 4 times the duplicate will be counted twice:
1. icao == "XXX" -> flag = false
2. icao == "XXX" and last == "XXX" and flag == false -> flag = true; count duplicate
3. icao == "XXX" and last == "XXX" and flag == true -> flag = false
4. icao == "XXX" and last == "XXX" and flag == false -> flag = true; count duplicate